modules/vsphere-problem-detector-running.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/using-vsphere-problem-detector-operator.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-running_{context}"]
+= Running the vSphere Problem Detector Operator checks
+
+You can override the schedule for running the checks and run the checks immediately.
```

Comment:
```suggestion
You can override the schedule for running the {operator-name} checks and run the checks immediately.
```

Reply:
Sounds good. Apologies for the careless error of missing the other `check`.